src/modules/fw_att_control/FixedwingAttitudeControl.cpp

```diff
@@ -288,11 +286,26 @@ void FixedwingAttitudeControl::Run()
 			parameters_update();
 		}
 
-		const float dt = math::constrain((att.timestamp - _last_run) * 1e-6f, 0.002f, 0.04f);
-		_last_run = att.timestamp;
+		float dt = 0.f;
+
+		static constexpr float DT_MIN = 0.002f;
```

Comment:
Would find it a bit cleaner to define it in the .hpp and use the new naming convention. See `kFlapSlewRate`

Reply:
Leave it up to you to address it prior merging or not.